static/js/popovers.js

```diff
@@ -251,6 +252,14 @@ function render_user_info_popover(
     load_medium_avatar(user, $(".popover-avatar"));
 }
 
+function get_user_groups_of_user(user_id) {
+    const user_groups_realm = user_groups.get_realm_user_groups();
+    const groups_of_user = user_groups_realm.filter((group) =>
+        user_groups.is_member_of(group.id, user_id),
+    );
+    return groups_of_user;
+}
```

Comment:
Let's make this a function in the `user_groups.js` library, and add some tests, as a prep commit.